devel/glib20/Makefile

```
@@ -34,6 +32,8 @@ MESON_ARGS=	-Db_lundef=false \
 		-Druntime_dir=/var/run \
 		-Dselinux=disabled \
 		-Dxattr=false
+
+CFLAGS+=	-DG_ENABLE_EXPERIMENTAL_ABI_COMPILATION
```

It looks like this is still required with the GLib branch from https://github.com/CTSRD-CHERI/glib/pull/2. However, we do not need this for all architectures.
```suggestion
CFLAGS_aarch64c+=	-DG_ENABLE_EXPERIMENTAL_ABI_COMPILATION
```

---

Fixed in the latest push.